src/stickers.yml

```diff
@@ -23,6 +23,287 @@
+  source: https://t.me/addstickers/PokeAnimated
+  tags:
+  - telegram
+  - pokemon
+  - for children
+
+585caf6cbbfd854f49bd734fb7f50441:
+  animated: true
+  key: 45bc33e475d881a55837e64f4ed65638f5b5fcf61e870c5cb4d7ac2888bcb121
+  source: https://t.me/addstickers/KangarooFighter
+  tags:
+  - telegram
+  - animals
+  - for children
+  - kangaroo
+
+67d3dfefcd64c994bd8f47c4ceb80548:
+  animated: true
+  key: ea874154ff436596e7ee6b6a30a047d52c897f70ebee129030e9cfc9aaac7915
+  source: https://t.me/addstickers/MrPugDog
+  tags:
+  - telegram
+  - animals
+  - pug
+  - dog
+  - for children
+
+71ae2a3f7c772f0948f4b9ee8d1d17ea:
+  animated: true
+  key: 544b290c0f949454ce5d1180ae54837658cbe9c32af211824de9589b9ee327f2
+  source: https://t.me/addstickers/StarPatrick
+  tags:
+  - telegram
+  - spongebob
+  - patrick
+  - patrick star
+  - for children
+
+7483b3e2302cad1a6a9c1237739427c8:
+  animated: true
+  key: c13027adae00438874043ad988f4a291eb029f8f55e3fbe1016e9741fff3064e
+  source: https://t.me/addstickers/MrSeal
+  tags:
+  - telegram
+  - animals
+  - seal
+  - for children
+
+803753d332b97d06aee34485882a5dc8:
+  animated: true
+  key: 83366449dc3147d1feccf307e1c41542ca9174ed6ced7137cbc11a0155478deb
+  source: https://t.me/addstickers/MiladyNoir
+  tags:
+  - telegram
+  - broken on desktop
+  - noir
+
+811a1fdaecc619a5be5f8423a6ca256f:
+  animated: true
+  key: e7d26e5ef6c7c4d57c2dbc806056f572533818393b5a8590b4466c9759158017
+  source: https://t.me/addstickers/Friendly_Panda
+  tags:
+  - telegram
+  - animals
+  - panda
+  - for children
+
+8a5f2c209203858a31c3f838074c726f:
+  animated: true
+  key: ab7e759fc6b45adfd151984e2a9c5b914f624e9684c08eb251930722a41c76d6
+  source: https://t.me/addstickers/RedShibaInu
+  tags:
+  - telegram
+  - doge
+  - meme
+  - dog
+  - animal
+  - shiba inu
+
+8b06294d286957bd35acb8b9da861598:
+  animated: true
+  key: 642c0c2357331d55669ab0178e4c933db4e9d24498777f445a605afc670fecff
+  source: https://t.me/addstickers/pusheen_animated_stickers
+  tags:
+  - telegram
+  - pusheen
+  - cat
+  - animals
+  - for children
+
+8f2233b92157c3598f5b0c5299996d18:
+  animated: true
+  key: 1ceb043beaf74f044104eac19ab3ebbea65770faf37736f63ef11ec1838f109b
+  source: https://t.me/addstickers/MrSeagull
+  tags:
+  - telegram
+  - animals
+  - seagull
+  - for children
+
+939fd93f5275de38f80ab423c3e4b87f:
+  animated: true
+  key: a700e417fba62195f6d1d64f2449e8092cc85c9c8840d000075c3a1445eab82e
+  source: https://t.me/addstickers/animated_animals
+  tags:
+  - telegram
+  - animals
+  - llama
+  - cat
+  - shark
+  - dog
+  - pig
+  - dragon
+  - rabbit
+  - turtle
+  - sheep
+  - panda
+
+9c2d7b466a08b20b829d26648f7c7e38:
+  animated: true
+  key: 1105adccef57d431818b93cc1436c5de0f805b9d59338eec2490f6065d0020d0
+  source: https://t.me/addstickers/TheVirus
+  tags:
+  - telegram
+  - virus
+  - corona
+  - broken on desktop
+
+aeff43e3014f56172e5e50f2766a2af3:
```

Comment:
This pack is a duplicate, please remove it (https://signalstickers.com/pack/9f85a3a9265faf61c08a69d948a3556a)

Reply:
Pumpkin removed.